internal/webhook/external/bitbucket.go

```diff
@@ -22,7 +23,8 @@ const (
 	bitbucketEventHeader     = "X-Event-Key"
 	bitbucketSignatureHeader = "X-Hub-Signature"
 
-	bitbucketPushEvent = "repo:push"
+	bitbucketPushEvent       = "repo:push"
+	bitbucketServerPushEvent = "repo:refs_changed" // Bitbucket Server (Data Center
```

Comment:
I could call this "bitbucketRefsChangedEvent" instead, but I felt it made sense to name it after the product.

Should I rename the `bitbucketPushEvent` to `bitbucketCloudPushEvent` for completeness too?

Reply:
I think I prefer `bitbucketRefsChangedEvent`. To provide additional context, you can add a comment detailing the discrepancy for Bitbucket Server; adding a link to the [payload schema](https://confluence.atlassian.com/bitbucketserver/event-payload-938025882.html).

Reply:
I renamed as follows:

* bitbucketServerPushEvent to bitbucketRefsChangedEvent
* The two structs (oddly capitalized) "bitBucketServerPushEvent" to "bitbucketRefsChangedEvent**Payload**"

And added a comment to each of these strings to indicate what they're used for.